src/message/messageActionSheet.js

```diff
@@ -246,6 +255,23 @@ export const constructMessageActionButtons = ({
   return buttons;
 };
 
+/** Returns the title for the action sheet */
+const getActionSheetTitle = (message: Message | Outbox, ownUser: User): string => {
+  // This means the message is a private message.
+  if (Array.isArray(message.display_recipient)) {
+    const recipients =
+      message.display_recipient.length === 1 && message.display_recipient[0].email === ownUser.email
+        ? message.display_recipient
+        : message.display_recipient.filter(r => r.email !== ownUser.email);
+    return recipients
+      .map(r => r.full_name)
+      .sort()
+      .join(', ');
```

Comment:
This logic should probably be unified with that in `src/webview/html/messageHeaderAsHtml.js`, rather than merely copied from it.

I'm not sure there's a good place for that to go at the moment, though; in the absence of somewhere better, I'd suggest a new file in `src/pm-conversations`. :confused: